Podcast/BookmarkTableViewCell.swift

```diff
@@ -171,7 +171,7 @@ class BookmarkTableViewCell: UITableViewCell {
     }
     
     func setPlayButtonToState(isPlaying: Bool) {
-        playButton.isEnabled = !isPlaying
+        playButton.isSelected = isPlaying
```

Comment:
Just checked and that bug is fixed with my latest commit. You can hold down the play button as long as you want and it'll play and update the UI correctly if you release from inside the button